src/renderer/modals/FullNode/steps/StepDisconnect.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React, { useCallback } from "react";
+import Box from "~/renderer/components/Box";
+import styled from "styled-components";
+import { Trans } from "react-i18next";
+import fullnodeIllustration from "~/renderer/images/disconnect.png";
+import Button from "~/renderer/components/Button";
+import { removeLSS } from "~/renderer/modals/FullNode/helpers";
+
+const Illustration = styled.div`
+  margin-bottom: 24px;
+  width: 236px;
+  height: 64px;
+  background: url(${fullnodeIllustration});
+  background-size: contain;
+  align-self: center;
+`;
+
+const Disconnect = () => (
+  <Box>
+    <Illustration />
+  </Box>
+);
+
+export const StepDisconnectFooter = ({ onClose }: { onClose: () => void }) => {
+  const onConfirmDisconnect = useCallback(() => {
+    removeLSS();
```

Comment:
- this is async
- this might fail

Reply:
So what do you want me to do about it?
If it fails to delete... then whelp, we tried 🤷

Reply:
yeah it's ok; we can keep it fire and forget